### DISCUSSION TITLE
What's the difference between jwilder/nginx-proxy and nginxproxy/nginx-proxy?

### DISCUSSION THREAD

**Discussion by omusil24** [nginx-proxy/nginx-proxy#2421] [Q&A] [answered]
Hi! I would like to know what's the difference between the jwilder/nginx-proxy and nginxproxy/nginx-proxy (Docker images)? Which one is more official or which one should be used primarily by new projects. Or are these separate projects?

And in case they're the same project, maybe it would be good to make a note about this in the readme file.

Thank you.

**Comment by SchoNie** [comment, marked as answer]
There is no difference, just the name. jwilder was the previous maintainer. Now buchdag is the maintainer and there was [chosen for a more common organisation he has control over. ](https://github.com/nginx-proxy/nginx-proxy/issues/1304#issuecomment-811851949)

In discussion https://github.com/nginx-proxy/nginx-proxy/discussions/2106 you can read about the name change.

For backwards compatibility the same image is pushed to both [jwilder/nginx-proxy](https://hub.docker.com/r/jwilder/nginx-proxy) and [nginxproxy/nginx-proxy](https://hub.docker.com/r/nginxproxy/nginx-proxy) registries so people using the 'old' name did not had to change.

But the images are identical in both as you can see they have the same digest.

The comment from @buchdag from https://github.com/nginx-proxy/nginx-proxy/discussions/2280#discussioncomment-6564106 also explains why 2 registries have to exist:
> Unfortunately you can't transfer and/or rename Dockerhub registries in the same way as Github repositories, so creating new registries under an org was pretty much the only way to have something we can control (we couldn't have asked jwilder to give us control over its private Dockerhub account either, I just have a push token to jwilder/nginx-proxy and jwilder/docker-gen).

> If you're curious you can take a look [at the build workflow](https://github.com/nginx-proxy/nginx-proxy/blob/main/.github/workflows/build-publish.yml#L47-L49), it's the exact same image built once and tagged with three different names (we also publish the image to Github container registry).
